config/routes.rb

```diff
@@ -2,12 +2,16 @@
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   namespace :api do
     namespace :v1 do
-      # Sensors
-      get '/users/:id/gardens/:id/sensors', to: 'sensors#index'
-      get '/users/:id/gardens/:id/sensors/:id', to: 'sensors#show'
-      post '/users/:id/gardens/:id/sensors', to: 'sensors#create'
-      patch '/users/:id/gardens/:id/sensors/:id', to: 'sensors#update'
-      delete '/users/:id/gardens/:id/sensors/:id', to: 'sensors#delete'
+      resources :users do
+        resources :gardens do
+          # Sensors
+          get '/sensors', to: 'sensors#index'
+          get '/sensors/:id', to: 'sensors#show'
+          post '/sensors', to: 'sensors#create'
+          patch '/sensors/:id', to: 'sensors#update'
+          delete '/sensors/:id', to: 'sensors#delete'
+        end
+      end
     end
```

Comment:
This could look cleaner if you wanted to use `resources :sensors`

Reply:
Great Suggestion! It's up now